tests/resources/quandl_samples/rebuild_samples.py

```diff
@@ -2,62 +2,77 @@
 Script for rebuilding the samples for the Quandl tests.
 """
 from __future__ import print_function
-from operator import methodcaller
 
-import pandas as pd
 import requests
+from io import BytesIO
+from zipfile import ZipFile
+from six.moves.urllib.parse import urlencode
+from zipline.testing import test_resource_path, write_compressed
 
+QUANDL_DATA_URL = (
+    'https://www.quandl.com/api/v3/datatables/WIKI/PRICES.csv?'
+)
 
-from zipline.data.quandl import format_wiki_url
-from zipline.testing import test_resource_path, write_compressed
 
+def format_table_query(api_key,
+                       start_date,
+                       end_date,
+                       symbols):
+    query_params = [
+        ('api_key', api_key),
+        ('date.gte', start_date),
+        ('date.lte', end_date),
+        ('ticker', ','.join(symbols)),
+    ]
+    return (
+        QUANDL_DATA_URL + urlencode(query_params)
+    )
 
-def zipfile_path(symbol):
-    return test_resource_path('quandl_samples', symbol + '.csv.gz')
+
+def zipfile_path(file_name):
+    return test_resource_path('quandl_samples', file_name)
 
 
 def main():
-    start_date = pd.Timestamp('2014')
-    end_date = pd.Timestamp('2015')
-    symbols = 'AAPL', 'MSFT', 'BRK_A', 'ZEN'
-    names = (
-        'Apple Inc.',
-        'Microsoft Corporation',
-        'Berkshire Hathaway Inc. Class A',
-        'Zendesk Inc',
-    )
-    print('Downloading equity data')
-    for sym in symbols:
-        url = format_wiki_url(
-            api_key=None,
-            symbol=sym,
-            start_date=start_date,
-            end_date=end_date,
-        )
-        print('Fetching from %s' % url)
-        response = requests.get(url)
-        response.raise_for_status()
+    api_key = 'Replace with Quandl API key.'
```

Comment:
Might be worth documenting or linking to how to get an API key. On a less important note, parameterizing this so that the user can pass the API key from the commandline would be helpful.